src/components/ArticlePage.jsx

```diff
@@ -24,16 +28,53 @@ function ArticlePage({topic}){
         return <Article article={article} topic={topic}/>
     }
 
+    function handleCommentButton(){
+        setCommentClick(true)
+    }
+
+    function RenderCommentButton(){
+            return (
+                <button onClick={handleCommentButton}>comment on this post</button>
+            )
+
+    }
+
+    function RenderPostComment(){
+        if(commentClick === true){
+            return (
+                <div> 
+                    <form onSubmit={handlePostComment}>
+                        <label >Comment:</label>
+                        <input type="text" name="body"/>
+                        <input type="submit" value="Submit"></input>
+                    </form>
+                </div>
+            )
+        }
+    }
+
+    function handlePostComment(event){
+        const commentBody = event.target.body.value;
+        axios.post(`https://nc-final-project.onrender.com/api/articles/${article_id}/comments`, {username: user.user, body: commentBody})
+        .then(({data: {postedComment}}) => {
+            console.log(postedComment)
```

Comment:
This is supposed to be a final code that you send in for review and hopefully accepted as part of the finished app. This means you should not commit any debugging logs in your code.